src/lavinmqctl.cr

```diff
@@ -234,18 +234,14 @@ class LavinMQCtl
       # Only used by tests in cloudamqp/rabbitmq-java-client
       @options["node"] = v
     end
-    @parser.on("-q", "--quiet", "suppress informational messages") do
-      @options["quiet"] = "yes"
-    end
-    @parser.on("-s", "--silent", "suppress informational messages and table formatting") do
```

Comment:
When I use it as a human, I like it to be super clear on what I am getting back.. do we remove that completely now? 
Maybe we could still have the informational messages, but they are suppressed when we format the output to json? 
like, `-f json` also sets `@options["quiet"] = "yes"`

Reply:
To be clear, the only thing that is removed are these messages:
```
puts "Listing queues for vhost #{vhost} ..." unless quiet?
puts "Listing connections ..." unless quiet?
puts "Closing connection #{name} ..." unless quiet?
puts "Listing policies for vhost #{vhost} ..." unless quiet?
puts "Listing exchanges for vhost #{vhost} ..." unless quiet?
```

Reply:
They are there for compatibility with rabbitmqctl

Reply:
added back the options and messages, java client tests now passes again 👍